tests/api/reports-import.php

```diff
@@ -151,4 +151,39 @@ public function test_import_params() {
 		$this->assertEquals( 'completed', $reports[0]['status'] );
 	}
 
+	/**
+	 * Test cancelling import actions.
+	 */
+	public function test_cancel_import() {
+		wp_set_current_user( $this->user );
+
+		// Populate all of the data.
+		$product = new WC_Product_Simple();
+		$product->set_name( 'Test Product' );
+		$product->set_regular_price( 25 );
+		$product->save();
+
+		$order = WC_Helper_Order::create_order( 1, $product );
+		$order->set_status( 'completed' );
+		$order->set_date_created( time() - ( 3 * DAY_IN_SECONDS ) );
+		$order->save();
+
+		// Cancel outstanding actions.
+		$request  = new WP_REST_Request( 'POST', $this->endpoint . '/cancel' );
+		$response = $this->server->dispatch( $request );
+		$report   = $response->get_data();
+
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertEquals( 'success', $report['status'] );
+		WC_Helper_Queue::run_all_pending();
```

Comment:
Shouldn't the queue be empty now? Do we need to call this?

Reply:
Removed in b8e8f99.